symplyphysics/laws/kinematic/vector/velocity_relative_to_reference_frame.py

```diff
@@ -0,0 +1,66 @@
+from sympy import Expr, symbols
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    Quantity,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+)
+from symplyphysics.core.vectors.arithmetics import (
+    diff_cartesian_vector_components,
+    integrate_cartesian_vector_components,
+)
+
+# Description
+## For any reference frame, whether it is inertial or not, the motion relative to it can be described using
+## the position vector relative to that frame's origin.
+
+# Law: v_rel = (dx/dt, dy/dt, dz/dt)
```

Comment:
You're right, they are similar. Well, this is the continuation of the topic about absolute/relative/transfer velocities I'm implementing right now. `relative`, because the coordinates of the point are relative to some frame of reference.

Reply:
Good point. I think there is some intuition to use `velocity` without additional characteristics when talking about relative velocity. So IMO it is better to have it as a definition of velocity and add a note about other types of velocities in description.

Reply:
What bothers me is that this law is tied to the Cartesian coordinate system and it would require separate laws for other coordinate systems. Maybe we could make this in such a way that it is obtained as a property of a coordinate system rather than through separate laws.

Reply:
You can write it as `v_rel = dr / dt`, where r is position vector. Now it does not rely on cartesian coordinates and also fits better with `diff_cartesian_vector` function.
Yet, we do not have general vector derivative available. So it will be limited to cartesian coordinates anyway.

Reply:
I mean there are already several formulas available for different coordinate systems, whose most basic form is $`\vec v_\text{rel} = \sum_i h_i \dot x_i \vec e_i`$ where $`h_i`$ is the $`i`$<sup>th</sup> Lamé coefficient corresponding to the $`i`$<sup>th</sup> coordinate. So we only need to define (probably in `core`) the Lamé coefficients.

Reply:
It's interesting option for a law. But I doubt we will see it anytime soon.
We may eventually try to use Lame coefficients for coordinate transformations though.